docs/user-guide/featured.md

```diff
@@ -0,0 +1,29 @@
+## Featured Notebooks & Collections
+
+```eval_rst
+.. meta::
+   :description: Featured Notebooks and Collections.
+```
+
+<a href="https://jovian.ai?utm_source=docs" target=_blank> Jovian </a> provides you with the feature to highlight some of your Notebooks & Collections on profile overview. You can feature like those
+related to your project , or some of your favourite topics ,or frameworks/languages used, or other important qualities that you find useful on your profile.
+
+### How to add Notebooks/Collections in Featured Section?
+
+1. For each notebook and collection you have been provided with the option to Pin it.
+   Note that you cannot pin private/secret notebooks and private collections.
```

Comment:
`Sphinx-rtd-theme` (theme that we are using for our docs page) has support for this kind of section. 

You can add `Note` with this syntax
```
```eval_rst
.. note:: {text}
```
**Preview**
![image](https://user-images.githubusercontent.com/24864161/97273858-dd5dcc80-1859-11eb-90ef-806e56a8c310.png)

It been used in `verison.md` check it out for reference.